mousestyles/visualization/plot_classification.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def plot_performance(model):
+    """
+    Plots the performance of classification model.It
+    is a side-by-side barplot. For each strain, it plots
+    the precision, recall and F-1 measure.
+    Parameters
+    ----------
+    model: string
+           The model used to classify the strain
+    Returns
+    -------
+    None
+    """
+    if model is 'SVM':
```

Comment:
I think we should not load file here. We should take the result npy as an argument here. And in plots folder, we then write a script to get different models' npy.